docs/en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md

```diff
@@ -199,6 +199,72 @@ convention in GraphQL APIs to use lowerCamelCase fields, so this is given by def
 
 [/notice]
 
+### Bulk loading models
+
+It's likely that in your application, you have a whole collection of classes you want exposed to the API, with roughly
+the same fields and operations exposed on them. It can be really tedious to write a new declaration for every single
+dataobject in your project, and as you add new ones, there's a bit of overhead in remembering to add it to the
+GraphQL schema.
+
+Common use cases might be:
+
+* Add everything in `App\Models`
+* Add every implementation of `BaseElement`
+* Add anything with the `Versioned` extension
+* Add everything that matches `src/*Model.php`
+
+You can create logic like this using the `bulkLoad` configuration file, which allows you to specify groups of directives
+that load a bundle of classes and apply the same set of configuration to all of them.
+
+
+By default, four loaders are provided to you to help gather specific classnames:
+
+* `namespaceLoader`: Fuzzy match the namespace of the model
+* `inheritanceLoader`: Get everything that matches or extends a given base class
+* `extensionLoader`: Get any class that has a given extension applied
+* `filepathLoader`: Get any classes in files matching a given glob expression.
+
+Each block starts with a collection of all classes that gets filtered as each loader runs. The primary job
+of a loader is to _remove_ classes from the entire collection, not add them in.
```

Comment:
Examples would definitely help here!

Reply:
Lol, there's a huge block of example code that I had in the clipboard and never put back in. Good catch..🤣